sdo/eclipselink.sdo.test.server.maven/src/assembly/eclipselink.sdo.test.server.ear.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this belongs under 'src/test/assembly' folder (or 'src/main/assembly')

Reply:
Yes, may be similar change will be in all JPA test modules.

Reply:
Done. Most assembly files are in the 'src/it/assembly' directory.
it...like integration tests